libs/mrwebrtc-unityplugin/include/api.h

```diff
@@ -12,6 +12,7 @@ extern "C" {
 
 using mrsResult = Microsoft::MixedReality::WebRTC::Result;
 using mrsNativeVideoHandle = mrsObjectHandle;
+typedef void (*TextureSizeChangeCallback)(int width, int height, mrsRemoteVideoTrackHandle handle); // would like to know how to put this in native_renderer.h
```

Comment:
Can you please rename that one, at least with `mrs` prefix, and ideally also with `Changed` instead of `Change` for consistency with callback naming? Thanks!